2024-12-03 00:42:46,442 - root - INFO - => building model
2024-12-03 00:42:48,845 - root - INFO - FLOPs: 4.58 G, Params: 19.49 M
Traceback (most recent call last):
  File "/ihome/hkarim/yip33/IDL_11785_Project/train.py", line 232, in <module>
    main()
  File "/ihome/hkarim/yip33/IDL_11785_Project/train.py", line 83, in main
    wandb.log({"FLOPs (G)": flops / 1e9, "Params (M)": params / 1e6})
  File "/ix1/hkarim/yip33/custom_miniconda/envs/cvt_env/lib/python3.9/site-packages/wandb/sdk/lib/preinit.py", line 36, in preinit_wrapper
    raise wandb.Error(f"You must call wandb.init() before {name}()")
wandb.errors.errors.Error: You must call wandb.init() before wandb.log()
srun: error: gpu-n43: task 0: Exited with exit code 1
